pkg/util/admission/grant_coordinator.go

```diff
@@ -1014,3 +1014,25 @@ func (e *ElasticCPUGrantCoordinator) close() {
 func (e *ElasticCPUGrantCoordinator) tryGrant() {
 	e.elasticCPUGranter.tryGrant()
 }
+
+// PacerMaker is used to construct a new admission.Pacer.
+//
+// PacerMaker interface should be used in the SQL layer in lieu of
+// ElasticCPUGrantCoordinator because the ElasticCPUGrantCoordinator
+// implements several features which are irrelevant to SQL.
+type PacerMaker interface {
```

Comment:
This sounds so medical!

Reply:
I was resisting the urge to call it `PaceMaker`.